<launch>
	<node name="hokuyo_node" pkg="hokuyo_node" type="hokuyo_node"/>
	<param name="calibrate_time" value="false" />
	<param name="port" value="/dev/ttyACM0" />


	<node name="my_vfh_test_node" pkg="ukftest" type="my_vfh_test_node"/>

	<node name="serial_to_uav_node" pkg="serial_to_uav" type="serial_to_uav_node" />
	<param name="is_write_on" value="true" />


	<node name="laser_neg_estimation_node" pkg="ukftest" type="laser_neg_estimation_node"/>


	<node name="uav_neg_laser_controller_node" pkg="ukftest" type="uav_neg_laser_controller_node" output="screen"/>
	<!-- PID parameters -->
	<param name="Kp_pos" value="1.0" />
	<param name="Ki_pos" value="0.0" />
	<param name="Kd_pos" value="0.0" />

	<param name="Kp_vel" value="1.20" />
	<param name="Ki_vel" value="0.20" />
	<param name="Kd_vel" value="0.34" />

	<param name="pid_gain" value="1.0" />
	<param name="controlLimit" value="2.0" />
</launch>


